docs/getting_started/faq.rst

```diff
@@ -135,8 +136,8 @@ This can also be done with:
     modin_cfg.Engine.put('unidist') # Modin will use Unidist
     unidist_cfg.Backend.put('mpi') # Unidist will use MPI backend
 
-We also have an experimental HDK-based engine of Modin you can read about :doc:`here </development/using_hdk>`.
-We plan to support more execution engines in future. If you have a specific request,
+We also have an experimental HDK-based engine of Modin. You can read about on :doc:`Using HDK </development/using_hdk>`
```

Comment:
```suggestion
We also have an experimental HDK-based engine of Modin you can read about on :doc:`Using HDK </development/using_hdk>`
```

Reply:
I divided it on purpose. Multiple subjects and independent clauses.

Reply:
I think we can leave either

```
We also have an experimental HDK-based engine of Modin, you can read about on :doc:`Using HDK </development/using_hdk>`
```
or

```
We also have an experimental HDK-based engine of Modin. You can read about it on :doc:`Using HDK </development/using_hdk>`
```

Reply:
ok